tox.ini

```diff
@@ -27,6 +27,7 @@ addopts =
     --cache-clear
     --cov .
     --cov-report term-missing
+    --log-level DEBUG
 filterwarnings =
     ignore:cannot collect test class 'Test.*' because it has a __init__ constructor:pytest.PytestCollectionWarning
     ignore:Using or importing the ABCs:DeprecationWarning:botocore
```

Comment:
```suggestion
filterwarnings =
    ignore:cannot collect test class 'Test.*' because it has a __init__ constructor:pytest.PytestCollectionWarning
    ignore:Using or importing the ABCs:DeprecationWarning:botocore
log_level = DEBUG
```